src/components/pages/user/tabs-content/profile-tab-content.tsx

```diff
@@ -24,30 +26,47 @@ const ProfileTabContent: React.FC<any> = () => {
       <ItemWrapper title="Avatar">
         <AvatarForm avatarUrl={viewer.avatar_url} />
       </ItemWrapper>
-      <ItemWrapper
-        title={
-          isConnectedToGithub
-            ? 'Your Account is Connected to Github'
-            : 'Connect to GitHub'
-        }
-      >
-        {isConnectedToGithub ? (
-          <p>You are able to login to egghead using your Github account!</p>
-        ) : (
-          <>
-            <p>Connect your GitHub account to log in with GitHub Oauth.</p>
-            <div className="mt-2">
-              <GithubConnectButton authToken={authToken} />
-            </div>
-          </>
-        )}
-      </ItemWrapper>
+      <ConnectGithub
+        isConnectedToGithub={isConnectedToGithub}
+        authToken={authToken}
+      />
     </div>
   )
 }
 
 export default ProfileTabContent
 
+const GithubDisConnectButton: React.FunctionComponent<{
```

Comment:
I think the `C` in `DisConnect` should be lowercase (otherwise, this PR looks great to me 🙌)